### PR TITLE
MGMT-1621 - update ignition config to 4.6: do not set provisioing int…

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**
…erface, set provisionig network to manage, remove unot-set fields

### CODE REVIEW

File: internal/installcfg/installcfg.go [hunk 1]
```diff
@@ -12,27 +11,19 @@ import (
...
+	BootMode       string `yaml:"bootMode"`
 }
 
 type baremetal struct {
-	ProvisioningNetworkInterface string `yaml:"provisioningNetworkInterface"`
-	APIVIP                       string `yaml:"apiVIP"`
-	IngressVIP                   string `yaml:"ingressVIP"`
-	DNSVIP                       string `yaml:"dnsVIP"`
-	Hosts                        []host `yaml:"hosts"`
+	ProvisioningNetwork string `yaml:"provisioningNetwork"`
```
Comment (tsorya) [inline internal/installcfg/installcfg.go:22]: just to understand, previously we added interface name? or what? and now ip?

File: internal/installcfg/installcfg.go [hunk 2]
```diff
@@ -150,61 +149,39 @@ func getBasicInstallConfig(cluster *common.Cluster) *InstallerConfigBaremetal {
...
-		} else {
-			hosts[i].Name = fmt.Sprintf("openshift-master-%d", masterCount)
-			hosts[i].Role = string(models.HostRoleMaster)
-			masterCount += 1
-		}
-		hosts[i].Bmc = bmc{
-			Address:  fmt.Sprintf("ipmi://192.168.111.1:%d", dummyPort+i),
-			Username: "admin",
-			Password: "rackattack",
+	yamlHostIdx := 0
```
Comment (tsorya) [inline internal/installcfg/installcfg.go:158]: why this var is needed? range is not returning index?

File: internal/installcfg/installcfg.go (hunk 1 above)
Comment (yevgeny-shnaidman) [reply internal/installcfg/installcfg.go:22]: we had interface name, now we won't need it, we just need to define the type of the provisioning network: "unmanaged", "disabled"

File: internal/installcfg/installcfg.go (hunk 2 above)
Comment (yevgeny-shnaidman) [reply internal/installcfg/installcfg.go:158]: we are skipping over disabled

Comment (tsorya) [reply internal/installcfg/installcfg.go:158]: i see it thanks

**Pull request merged**